packages/features/form-builder/FormBuilder.tsx

```diff
@@ -201,7 +201,7 @@ export const FormBuilder = function FormBuilder({
                       // It has the same drawback that if the label is changed, the value of the option will change. It is not a big deal for now.
                       value.splice(index, 1, {
                         label: e.target.value,
-                        value: e.target.value.toLowerCase().trim(),
```

Comment:
Routing Form uses the strategy of not lowercasing the option label and using it as is. 
Changed this logic here to match that, otherwise an option value forwarded from Routing Form won't be prefillable in Booking Form.